libc/calls/metalfile.internal.h

```diff
@@ -9,6 +9,14 @@ struct MetalFile {
   size_t pos;
 };
 
+extern void *__ape_com_base;
+extern size_t __ape_com_size;
+
 COSMOPOLITAN_C_END_
 #endif /* !(__ASSEMBLER__ + __LINKER__ + 0) */
+
+#define APE_COM_NAME     "/ape.com"
+#define APE_COM_ALT_NAME "/proc/self/exe"
+#define APE_COM_URI      "file:/proc/self/exe"
```

Comment:
Why do we need a URI?

Reply:
> Why do we need a URI?

@jart : I was following the `tool/build/lib/elfwriter_zip.c` behavior of using a URI as an ELF symbol which can be exported (or imported) as part of an ABI.  But other than that, there is probably no real need for a URI here.  Thank you!